ui/src/test/java/edu/wpi/grip/ui/pipeline/AddSourceButtonTest.java

```diff
@@ -93,6 +94,7 @@ public void testClickOnCreateNetworkTableOpensDialog() throws Exception {
     }
 
     @Test
+    @Ignore("Fails on all platforms") // TODO: Figure out why this is broken.
```

Comment:
At minimum, please at least ensure that this actually works when driven from the actual UI.
Same with the one below.

Reply:
I don't know enough about the UI to know what this test even does? Do you know?

Reply:
The test is failing because adding a webcam source is broken.

You can recreate this bug by running the GRIP UI and selecting "Add Source" -> "Webcam". Select your camera (on my computer this is 0). And press OK. GRIP will crash with a JVM error.
```
Process 'command '/Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home/bin/java'' finished with non-zero exit value 134
```